ibex_configs.yaml

```diff
@@ -20,6 +21,16 @@ small-3cmult:
 maxperf-1cmult:
   RV32E                    : False
   RV32M                    : True
+  RV32B                    : False
+  BranchTargetALU          : True
+  WritebackStage           : True
+  MultiplierImplementation : "single-cycle"
+
+# maxpref-1cmult config above with bitmanip extension
```

Comment:
Since we're planning to advertise these configurations as "supported configurations": Maybe add an explicit "experimental-" or something like that to the config names? (I know we don't do that for the third pipeline stage currently, but at least there the implementation is complete, while for bitmanip we're very early and very incomplete at this point.)

Reply:
As we hadn't really documented much of this yet or official put down what it means to be 'supported' I was intentionally being quite vague.

I'll put both the maxperf configurations under a comment calling them 'EXPERIMENTAL CONFIGURATIONS', and add 'experimental-' as you suggest. That along with the other EXPERIMENTAL labels and lack of documentation should give the correct impression for now.